src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/tests/test_acr_commands.py

```diff
@@ -85,8 +84,8 @@ def test_acr_create_with_managed_registry(self, resource_group, resource_group_l
 
     @ResourceGroupPreparer(location='eastus')
     def test_acr_create_webhook(self, resource_group, resource_group_location):
-        registry_name = self.create_random_name('clireg', 50)
-        webhook_name = self.create_random_name('cliregwebhook', 50)
+        registry_name = self.create_random_name('clireg', 20)
+        webhook_name = 'cliregwebhook'
```

Comment:
If the webhook name is globally unique it must be set to a random name, otherwise this looks fine.

Reply:
It's unique within each registry.